src/algebra/ordered_ring.lean

```diff
@@ -1024,11 +1073,12 @@ begin
   simp [some_eq_coe, coe_mul.symm, mul_assoc]
 end
 
-private lemma one_mul' : ∀a : with_top α, 1 * a = a
+-- FIXME is `nontrivial` really needed here?
```

Comment:
```suggestion
```

Yes, because `0 * ⊤ = 0` (by definition, see `with_top.mul_def`).

Reply:
Could you address this comment?